app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class PaymentsController < ApplicationController
+  before_action :set_product
+
+  def new; end
+
+  def create
+    customer = Stripe::Customer.create(email: current_user.email)
+    token = Stripe::Token.create(card: { number: params[:number],
```

Comment:
What do you think about moving this code to service?
You have the same code(token creating) in another controller